scm-webapp/src/main/java/sonia/scm/lifecycle/modules/ScmSecurityModule.java

```diff
@@ -114,14 +117,24 @@ protected void configureShiroWeb()
     }
 
     // bind constant
-    bindConstant().annotatedWith(Names.named("shiro.loginUrl")).to(
-      "/index.html");
+    bindConstant().annotatedWith(Names.named("shiro.loginUrl")).to("/index.html");
 
     // disable access to mustache resources
     addFilterChain("/**.mustache", filterConfig(ROLES, "nobody"));
 
     // disable session
+    disableSession();
+  }
+
+  private void disableSession() {
     addFilterChain("/**", NO_SESSION_CREATION);
+    bindConstant().annotatedWith(Names.named("shiro.sessionStorageEnabled")).to(false);
+
+    DefaultSubjectDAO subjectDAO = new DefaultSubjectDAO();
+    DefaultSessionStorageEvaluator sessionStorageEvaluator = new DefaultSessionStorageEvaluator();
+    sessionStorageEvaluator.setSessionStorageEnabled(false);
+    subjectDAO.setSessionStorageEvaluator(sessionStorageEvaluator);
+    bind(SubjectDAO.class).toInstance(subjectDAO);
   }
```

Comment:
I thought SCM-Manager 2 doesn't use server sessions. So what is this actually for?